openapi/publicapi_spec.json

```diff
@@ -1552,6 +1552,135 @@
           }
         }
       },
+      "CardPayment" : {
```

Comment:
GetPaymentResult included a "_links" property. Looks like this was only used fro the API spec. Can we use PaymentWithAllLinks for the API spec or is there some reason this doesn't work?